src/main/scala/com/databricks/spark/avro/SchemaConverters.scala

```diff
@@ -98,14 +119,26 @@ private object SchemaConverters {
       recordNamespace: String): T = {
     val fieldsAssembler: FieldAssembler[T] = schemaBuilder.fields()
     structType.fields.foreach { field =>
-      val newField = fieldsAssembler.name(field.name).`type`()
-
-      if (field.nullable) {
-        convertFieldTypeToAvro(field.dataType, newField.nullable(), field.name, recordNamespace)
-          .noDefault
-      } else {
-        convertFieldTypeToAvro(field.dataType, newField, field.name, recordNamespace)
-          .noDefault
+      breakable {
```

Comment:
Yes, it is nice to check them before. But as far as simplicity and readability goes...
```scala
breakable {
  if (predicate) break
  body
}
```
is equivalent to
```scala
if (!predicate) {
  body
}
```
Hence is is shorter, cleaner and other folks do not need to know what breakable is. Right?

BTW: the code here can be written as...
```scala
val nonAliasStructFields = structType.fields.filterNot(field =>
  field.metadata.contains(METADATA_KEY_PARENT) && !field.metadata.getString(METADATA_KEY_PARENT).equals(field.name))
nonAliasStructFields.foreach { field =>
  var newFieldBuilder = fieldsAssembler.name(field.name)
...
```